platform/security/certificate/security-ocsp-checker/src/main/java/org/codice/ddf/security/ocsp/checker/OcspChecker.java

```diff
@@ -446,6 +474,191 @@ private void postErrorEvent(String errorMessage) {
     LOGGER.debug(errorMessage);
   }
 
+  private void logRequest(OCSPReq ocspRequest) {
+    StringBuffer logbuff = new StringBuffer();
+    logbuff.append("OCSP Request:\n");
+    logbuff.append("  TBSRequest:\n");
+    logbuff.append("    version: " + getValueOrDefault(ocspRequest.getVersionNumber(), "") + "\n");
+    logbuff.append(
+        "    requestorName: "
+            + getValueOrDefault(ocspRequest.getRequestorName(), "").toString()
+            + "\n");
+    logbuff.append("    requestList:\n");
+    Req[] requests = ocspRequest.getRequestList();
+    if (requests != null) {
+      int i = 0;
```

Comment:
✏️ If we need a counter for the logs then we may just want to use a regular index-based `for` loop.

Reply:
Fixed.